recipes/tktable/license.txt

```diff
@@ -0,0 +1,41 @@
+		   * COPYRIGHT AND LICENSE TERMS *
```

Comment:
You can delete this file since it is in the tarball

Reply:
@isuruf here the corresponding PR, please review, thanks! https://github.com/conda-forge/tktable-feedstock/pull/1